src/mlpack/methods/ann/layer/convolution.hpp

```diff
@@ -301,8 +301,7 @@ class ConvolutionType : public Layer<MatType>
    * @param output The rotated output.
    */
   template<typename CubeType>
-  void Rotate180(const CubeType& input, CubeType& output,
-      const typename std::enable_if_t<IsCube<CubeType>::value>* = 0)
+  void Rotate180(const CubeType& input, CubeType& output)
```

Comment:
If `ConvolutionType` has a `CubeType` typedef already, then isn't the `CubeType` template parameter for this function unnecessary?  I see in `GroupedConvolutionType` the template parameter is not there; it should probably be the same here. :+1:

Reply:
Yes, this was added, before adding the `CubeType` and then I forgot about it completely